arviz/tests/base_tests/test_plots_bokeh.py

```diff
@@ -148,6 +148,69 @@ def test_plot_density_bad_kwargs(models):
     with pytest.raises(ValueError):
         plot_density(obj, hdi_prob=2, backend="bokeh", show=False)
 
+def combinedims_test_plot_density_float(models, kwargs):
+    obj = [getattr(models, model_fit) for model_fit in ["model_1", "model_2"]]
+    axes = plot_density(obj, combine_dims=['school'], backend="bokeh", show=False, **kwargs)
+    assert axes.shape[0] >= 6
+    assert axes.shape[0] >= 3
```

Comment:
The 2nd assert is unnecessary. I don't understand however why are there more than 6 subplots, I think there should only be 4, one per variable: mu, tau, theta and theta_tilde. I also don't understand how is this test passing when `kwargs` is not defined anywhere :confused:

Reply:
Ohhhh, I see now. Pytest only executes the functions that start with `test_`, as the new tests start with `combineddims` they are not executed.

Reply:
I've changed all `combine_dims`-related test cases to follow the correct format. I have also updated the `assert` conditions to be `assert axes.shape[0] == 4` and added the pytest decorator to "pass in" any other arguments we may need, so `kwargs` should not be undefined now. The tests do all pass, but may need to be inspected for redundancy, per your previous comment. 

Notably, I have taken out all the new Bokeh tests for simplicity; once we receive feedback, I'll add them in, since they should 
1) follow a similar structure as the existing tests
2) test the same functionality as the Matplotlib tests.

Please let us know if we're missing something key, and thanks for the detailed feedback!